Engines/Wine/QuickScript/GoG Script/script.js

```diff
@@ -0,0 +1,70 @@
+include(["engines", "wine", "quick_script", "quick_script"]);
+include(["utils", "functions", "net", "download"]);
+include(["engines", "wine", "engine", "object"]);
+
+function GogScript() {
+    QuickScript.call(this);
+}
+
+GogScript.prototype = Object.create(QuickScript.prototype);
+
+GogScript.prototype.constructor = GogScript;
+
+GogScript.prototype.gogSetupFileName = function(setupFileName) {
+    this._setupFileName = setupFileName;
+    return this;
+}
+
+/**
+ * Presents a Gog.com login window to the user, login to its account and return a token that can be used later.
+ * Stores the tocken in a parameter
+ * @param {SetupWizard} setupWizard The setupWizard to use
+ * @returns {GogScript} This
+ */
+GogScript.prototype.loginToGog = function (setupWizard) {
+    var browserWindow = setupWizard.createBrowser("Please login to your GoG.com account so that we can download the game for you:");
+    browserWindow.goToUrl("https://auth.gog.com/auth?client_id=46899977096215655&redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient&response_type=code&layout=client2");
```

Comment:
Do we need all these URL parameters?

Reply:
Yes, absolutely :)
https://gogapidocs.readthedocs.io/en/latest/